Resources/Prototypes/_Mono/Shipyard/pluto.yml

```
@@ -0,0 +1,57 @@
+# SPDX-FileCopyrightText: 2023 IHAN <IHAN>
+# SPDX-FileCopyrightText: 2023 Kennedy
+# SPDX-FileCopyrightText: 2023 PECK
+# SPDX-FileCopyrightText: 2023 RealIHaveANameOfficial
+# SPDX-FileCopyrightText: 2024 Debug
+# SPDX-FileCopyrightText: 2024 Dvir
+# SPDX-FileCopyrightText: 2024 Maxtone
+# SPDX-FileCopyrightText: 2024 Salvantrix
+# SPDX-FileCopyrightText: 2024 Whatstone
+# SPDX-FileCopyrightText: 2025 Carolyn3114
+# SPDX-FileCopyrightText: 2025 Redrover1760
+# SPDX-FileCopyrightText: 2025 UnicornOnLSD
+# SPDX-FileCopyrightText: 2025 kyres1
+# SPDX-FileCopyrightText: 2025 starch
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+# Author Info
+# GitHub: UnicornOnLSD
+# Discord: unicornonlsd
+
+# Shuttle Notes:
+#
+- type: vessel
+  id: Pluto
+  parent: BaseVessel
+  name: DIS Pluto
+  description: A frigate sized cargo hauler manufactured by the Drake Industries shipyards. Fitted with a 25 type bluespace drive and more than enough space for small scale cargo hauling.
+  price: 19900
+  category: Small
+  group: Shipyard
+  shuttlePath: /Maps/_Mono/Shuttles/pluto.yml
+  guidebookPage: Null
+  class:
+  - Cargo
+  engine:
+  - Uranium
+
+- type: gameMap
+  id: Pluto
+  mapName: 'DIS Pluto'
+  mapPath: /Maps/_Mono/Shuttles/pluto.yml
+  minPlayers: 0
+  stations:
+    Twilight:
```

```suggestion
    Pluto:
```
This should be the ID of your BecomesStation component ID in your ship